doc/user_guide/electron_holography.rst

```diff
@@ -57,5 +58,38 @@ This option can be used for batch processing as the reconstruction_param can be
 
     >>> wave_image1 = im1.reconstruct_phase(rec_param=reconstruction_param)
 
-Further analyses of the reconstructed wave (phase) can be done using the :py:class:`~._signals.wave_image.ComplexSignal2D` class
-functionality (see dedicated documentation).
\ No newline at end of file
+Further analyses of the reconstructed wave (phase) can be done using :py:class:`~._signals.wave_image.ElectronWaveImage` class
+functionality (see bellow).
+
+
+The ElectronWaveImage class
+===========================
+
+The :py:class:`~._signals.wave_image.ElectronWaveImage` class can hold information about the complex electron
+wave. As such, relevant properties like the `amplitude`, `phase` and the `real` and `imag` part can be
+directly accessed and return appropriate :py:class:`~._signals.signal2d.Signal2D` signals.
+
+To transform a :py:class:`~._signals.complex_signal2d.ComplexSignal2D` (or subclass) into a
+:py:class:`~._signals.wave_image.ElectronWaveImage` use:
+
+.. code-block:: python
+
+    >>> im.set_signal_type('ElectronWaveImage')
```

Comment:
I didn't see that 1.0 is officially out until now, congratulations :-)!
Concerning the `signal_type` names, would 1.1 be okay for the change? Or should this be changed at all, due to backwards compatibility? It does not look that bad with the underscore, in my opinion. Alternatively `_signal_type` (the class properties) could be implemented as a list, so we can have a new and a legacy name...
I also saw that there are a few formatting problems with the user guide, I assume updating that would be okay, even between releases, right? I would submit a PR for that then.